tests/integration/client/room/roomAdmin.test.tsx

```diff
@@ -223,7 +223,7 @@ describe('Room Admin Modal Behavior', function () {
               );
               const submitButton = getByText('Submit');
               fireEvent.click(submitButton);
-              await findByText('Capacity is required to submit this form, and it must be a number.', { exact: false });
+              await findByText('Capacity is required to submit this form, and it must be a positive number.', { exact: false });
```

Comment:
As noted above, "positive" isn't necessary here.

Reply:
Also updated this in [c345f15](https://github.com/seas-computing/course-planner/pull/581/commits/c345f15902f6fb3fe68bce21766aa084e45a7520).